src/admin/client.ts

```diff
@@ -74,7 +76,7 @@ class DisplayOrder {
 
     public submit = () => {
         var msg = new Models.OrderRequestFromUI(this.side, this.price, this.quantity, this.timeInForce, this.orderType);
-        this._log.info("submitting order", msg);
```

Comment:
I'm not following the first paragraph "this is not related to the full space of the vmdk of docker...".

The logging is for development but if you'd like the ability to eliminate it, I'd prefer either

a) Changing it to Debug level then using logProvider (https://docs.angularjs.org/api/ng/provider/$logProvider) to turn it off
a) Wholly remove it, don't leave the commented version around.